MediaBrowser.Controller/Entities/Folder.cs

```diff
@@ -1008,7 +1007,7 @@ protected QueryResult<BaseItem> PostFilterAndSort(IEnumerable<BaseItem> items, I
 
             if (!string.IsNullOrEmpty(query.NameStartsWithOrGreater))
             {
-                items = items.Where(i => string.Compare(query.NameStartsWithOrGreater, i.SortName, StringComparison.CurrentCultureIgnoreCase) < 1);
+                items = items.Where(i => string.Compare(query.NameStartsWithOrGreater, i.SortName, StringComparison.OrdinalIgnoreCase) < 1);
```

Comment:
`CurrentCultureIgnoreCase` may be correct here

Reply:
I thought it might, but there's already a comparison on L1015 that uses `StringComparison.OrdinalIgnoreCase`